tests/test_build_recipe.py

```diff
@@ -196,7 +196,7 @@
   <description type="system">
     <author>SUSE LLC</author>
     <contact>https://www.suse.com/</contact>
-    <specification>SLE BCI DevelopmentContainer Test Container Image</specification>
+    <specification>SL BCI DevelopmentContainer Test Container Image</specification>
```

Comment:
Is SL BCI a thing? I thought that the shortest version was SUSE Linux BCI.

Reply:
changed.. lets see how the fallout behaves.